project/src/index.tsx

```diff
@@ -13,7 +12,9 @@ const root = ReactDOM.createRoot(
 root.render(
   <React.StrictMode>
     <App
-      cardsCount={Setting.CARDS_COUNT}
+      offers = {offers}
+      limit = {LIMIT_IMAGE}
+      reviews = {reviews}
```

Comment:
Константу для фото лучше сразу импортировать в компонент галереи
```suggestion
      offers={offers}
      reviews={reviews}
```